### PR TITLE
Fc 776 requisition list documentation

### PR DESCRIPTION
**Pull request by mo-fc**
Documented parts related to my work

### CODE REVIEW

File: source/docs/appendices/migration-guides.md [hunk 1]
```diff
@@ -23,6 +23,23 @@ To support [the Slider and Slider Magento2 page builder content types](/docs/mag
  };
 ```
 
+### `swatch` image format in base theme
+
+We backported the `swatch` image format from theme chocolatine to base theme. This image format is used to display product images of a requisition list. If you are using base theme and want to use the requisition list feature of the B2B module you have to add the swatch image format to your `src/config/images.js` as folows:
```
Comment (paul-vd) [inline source/docs/appendices/migration-guides.md:28]:
```suggestion
We backported the `swatch` image format from theme chocolatine to base theme. This image format is used to display product images of a requisition list. If you are using the base theme and want to use the requisition list feature of the B2B module you have to add the swatch image format to your `src/config/images.js` as follows:
```

File: source/docs/appendices/migration-guides.md [hunk 2]
```diff
@@ -35,27 +52,322 @@ In this release, we have improved the image resizing mechanism to be bit smarter
 };
 ```
 
+## Added requisition list features to B2B module (only theme chocolatine suported)
+
+If you are using base theme and wish to use B2B module you have to override `theme/modules/ProductView/Synthesis/AddProductToCart.js` from the base theme and provide a way to add to requisition list from the product page (see `<AddProductToRequisitionList>` and `modules/front-commerce-b2b/web/theme/modules/ProductView/Synthesis/AddProductToCart.js` for inspiration). You also need to override `theme/modules/AddToCart/_AddToCart.scss` if you have not already and just copy the styles from the base theme to it (`src/web/theme/modules/AddToCart/_AddToCart.scss`).
```
Comment (paul-vd) [inline source/docs/appendices/migration-guides.md:57]: Can you maybe add links to the references?
- `<AddProductToRequisitionList>`
- `modules/front-commerce-b2b/web/theme/modules/ProductView/Synthesis/AddProductToCart.js`

**Review by paul-vd — CHANGES_REQUESTED** [review]

File: source/docs/appendices/migration-guides.md [hunk 3]
```diff
@@ -35,27 +52,322 @@ In this release, we have improved the image resizing mechanism to be bit smarter
...
+
+If you are using theme chocolatine and have overided `theme/modules/ProductView/Synthesis/AddProductToCart.js` and want to use the B2B module you need to apply the folowing change to it:
```
Comment (real34) [inline source/docs/appendices/migration-guides.md:59]:
```suggestion
If you are using theme chocolatine and have overridden `theme/modules/ProductView/Synthesis/AddProductToCart.js` and want to use the B2B module you need to apply the following change to it:
```

File: source/docs/appendices/migration-guides.md [hunk 4]
```diff
@@ -35,27 +52,322 @@ In this release, we have improved the image resizing mechanism to be bit smarter
...
+      color: $fontColor;
+    }
+  }
+}
+```
+
+</details>
+<br/>
+
+if you overrode `theme/modules/AddToCart/_AddToCart.scss` and are using theme chocolatine apply the folowing diff to it:
```
Comment (real34) [inline source/docs/appendices/migration-guides.md:343]:
```suggestion
if you overrode `theme/modules/AddToCart/_AddToCart.scss` and are using theme chocolatine apply the following diff to it:
```

**Review by real34 — COMMENTED** [review]
Quick review for typos only now

File: source/docs/appendices/migration-guides.md (hunk 2 above)
Comment (mo-fc) [reply source/docs/appendices/migration-guides.md]: 0b9365cea93199a19913b3dd32e838f61dc983b9

**Review by paul-vd — APPROVED** [review]

**Review by real34 — APPROVED** [review]

File: source/docs/appendices/migration-guides.md [hunk 5]
```diff
@@ -35,27 +52,322 @@ In this release, we have improved the image resizing mechanism to be bit smarter
 };
 ```
 
+### Added requisition list features to B2B module (only theme chocolatine suported)
+
+If you are using base theme and wish to use B2B module you have to override `theme/modules/ProductView/Synthesis/AddProductToCart.js` from the base theme and provide a way to add to requisition list from the product page (see [AddProductToRequisitionList](https://gitlab.com/front-commerce/front-commerce/-/blob/2.12.0/modules/front-commerce-b2b/web/theme/modules/RequisitionList/AddProductToRequisitionList/AddProductToRequisitionList.js) and [B2B's AddProductToCart](https://gitlab.com/front-commerce/front-commerce/-/blob/2.12.0/modules/front-commerce-b2b/web/theme/modules/ProductView/Synthesis/AddProductToCart.js) for inspiration). You also need to override `theme/modules/AddToCart/_AddToCart.scss` if you have not already and just copy the styles from the base theme to it (`src/web/theme/modules/AddToCart/_AddToCart.scss`).
```
Comment (abailleulfc) [inline source/docs/appendices/migration-guides.md:57]:
```suggestion
If you are using base theme and wish to use the B2B module you have to override `theme/modules/ProductView/Synthesis/AddProductToCart.js` from the base theme and provide a way to add to requisition list from the product page (see [AddProductToRequisitionList](https://gitlab.com/front-commerce/front-commerce/-/blob/2.12.0/modules/front-commerce-b2b/web/theme/modules/RequisitionList/AddProductToRequisitionList/AddProductToRequisitionList.js) and [B2B's AddProductToCart](https://gitlab.com/front-commerce/front-commerce/-/blob/2.12.0/modules/front-commerce-b2b/web/theme/modules/ProductView/Synthesis/AddProductToCart.js) for inspiration). You also need to override `theme/modules/AddToCart/_AddToCart.scss` if you have not already and just copy the styles from the base theme to it (`src/web/theme/modules/AddToCart/_AddToCart.scss`).
```

File: source/docs/appendices/migration-guides.md [hunk 6]
```diff
@@ -35,27 +52,322 @@ In this release, we have improved the image resizing mechanism to be bit smarter
...
 @import "~theme/components/organisms/Tree/Tree";
+@import "~theme/modules/Company/CompanyStructure/CompanyStructure";
+@import "~theme/modules/RequisitionList/RequisitionList";
+@import "~theme/modules/RequisitionList/RequisitionListTable/RequisitionListTable";
+```
+
+If you overrode `src/web/theme/components/atoms/Button/_Button.scss` apply the following diff to it:
+
+<details>
+<summary>Diff for base theme</strong></summary>
```
Comment (abailleulfc) [inline source/docs/appendices/migration-guides.md:264]:
```suggestion
<summary><strong>Diff for base theme</strong></summary>
```

File: source/docs/appendices/migration-guides.md [hunk 7]
```diff
@@ -35,27 +52,322 @@ In this release, we have improved the image resizing mechanism to be bit smarter
...
+    }
++  }
+}
+```
+
+</details>
+<br/>
+
+<details>
+<summary>Diff for theme chocolatine</strong></summary>
```
Comment (abailleulfc) [inline source/docs/appendices/migration-guides.md:303]:
```suggestion
<summary><strong>Diff for theme chocolatine</strong></summary>
```

**Review by abailleulfc — CHANGES_REQUESTED** [review]

**Pull request merged**